### PR TITLE
Make configPath unable to update at runtime

### PR DESCRIPTION
**Pull request by RongtongJin**

### CODE REVIEW

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java [hunk 1]
```diff
@@ -775,13 +775,21 @@ private synchronized RemotingCommand updateBrokerConfig(ChannelHandlerContext ct
                 String bodyStr = new String(body, MixAll.DEFAULT_CHARSET);
                 Properties properties = MixAll.string2Properties(bodyStr);
                 if (properties != null) {
-                    LOGGER.info("updateBrokerConfig, new config: [{}] client: {} ", properties, ctx.channel().remoteAddress());
+                    LOGGER.info("updateBrokerConfig, new config: [{}] client: {} ", properties, callerAddress);
+
+                    if (properties.containsKey("brokerConfigPath")) {
+                        response.setCode(ResponseCode.SYSTEM_ERROR);
```
Comment (aaron-ai) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java:781]: Is `NO_PERMISSION` better than `SYSTEM_ERROR`?

File: controller/src/main/java/org/apache/rocketmq/controller/processor/ControllerRequestProcessor.java [hunk 2]
```diff
@@ -241,6 +241,12 @@ private RemotingCommand handleUpdateControllerConfig(ChannelHandlerContext ctx,
                 return response;
             }
 
+            if (properties.containsKey("configStorePath")) {
+                response.setCode(ResponseCode.SYSTEM_ERROR);
```
Comment (aaron-ai) [inline controller/src/main/java/org/apache/rocketmq/controller/processor/ControllerRequestProcessor.java:245]: Same here.

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java (hunk 1 above)
Comment (RongtongJin) [reply broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java]: done

File: controller/src/main/java/org/apache/rocketmq/controller/processor/ControllerRequestProcessor.java (hunk 2 above)
Comment (RongtongJin) [reply controller/src/main/java/org/apache/rocketmq/controller/processor/ControllerRequestProcessor.java]: > Same here.

done

**Review by fuyou001 — APPROVED** [review]
LGTM

**Review by lollipopjin — APPROVED** [review]
LGTM

**Review by aaron-ai — APPROVED** [review]

**Review by caigy — APPROVED** [review]

**Review by ShannonDing — APPROVED** [review]

**Comment by ljsquirrel** [comment]
Excuse me, I couldn't find any usage scenarios for configStorePathName in 4.9.6. Is the added filter configStorePath? Instead of configStorePathName.

**Pull request merged**